### PR TITLE
14.0 mig l10n fr ecotaxe

### PR DESCRIPTION
**Pull request by mourad-ehm**

### CODE REVIEW

**Comment by legalsylvain** [comment]
Hi @mourad-ehm could you rebase ?

**Review by ajaniszewska-dev — APPROVED** [review]

**Comment by ajaniszewska-dev** [comment]
@mourad-ehm  seems there are problems with merge, can you please check & eventually fix your PR?

File: l10n_fr_ecotaxe_purchase/tests/test_purchase_ecotaxe.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        self.assertEqual(self.purchase.amount_ecotaxe, 6.0)
+
+    def test_02_classification_weight_based_ecotaxe(self):
+        """ Tests multiple lines with weight based ecotaxes """
+        # in order to test the correct assignment of fixed ecotaxe
+        # I create a customer purchase.
+        partner12 = self.env.ref("base.res_partner_12")
+        purchase = self.create_purchase_partner(
+            purchase_amount=100.00, partner_id=partner12
+        )
```
Comment (ajaniszewska-dev) [inline l10n_fr_ecotaxe_purchase/tests/test_purchase_ecotaxe.py:65]: where `create_purchase_partner` is defined?

**Comment by gurneyalex** [comment]
@alexis-via I think the red build of ocabot on the branch is related to changes you are working on on OCA/edi 14.0 branch. Do you have an idea how this can be fixed?

**Comment by alexis-via** [comment]
The problems in OCA/edi v14 have been solved on Feb 14 2022 with this merge https://github.com/OCA/edi/pull/522

**Comment by gurneyalex** [comment]
Merge failing because we have a dependency somewhere on pykcs11 and this has no prebuilt wheel and a C extension requiring swig to build :/

**Pull request closed without merging**